modules/upgrade/pages/db-migration.adoc

```diff
@@ -84,161 +88,14 @@ The [path]``pg-migrate.sh`` script performs these operations:
 If the upgrade fails, the migration script will attempt to restore the database to its original state.
 ====
 
-
-
 .Procedure: Performing a Fast PostgreSQL Upgrade
 . Perform a database backup.
+Without a verified database backup, you must not initiate a fast upgrade.
```

Comment:
Again I'm amazed by the "you must not" :) I would have worded it as "you should not" or even "it is strongly recommended..."

Reply:
Yes, sometimes passive voice is better.

Reply:
We use passive voice in technical writing a lot, because the actor is often the least important information. So in this case, it's more about word order than active/passive. Here, we need the reader to notice this sentence if they don't have a backup, so we put that information first. If they have a backup, they can skip this sentence. If they don't, we need them to keep reading: do not initiate the upgrade.

As for the word choice, "should" is not as definite as "must". I often sit on the couch and say "I should clean the floors", but it doesn't mean I do. The same can be said for "it is strongly recommended". Aside from being far too wordy, it's still not as strong as "must". You don't get to argue with "must" ;)

All that said, it could be even clearer by saying "do not". This is both more concise, and more definite. But do you notice how I just removed the actor entirely? The actor is simply not important in this sentence, which further backs up the case for passive voice.

I wrote guidance on this type of thing here: https://github.com/SUSE/doc-susemanager/wiki/wug#language

Reply:
Thanks for the free English lesson :)

I was under the impression that - exactly because "must" is so strong - it should not be used in English as it can be perceived as impolite and one should use something like "you might consider to do this and that" instead.

The British are too polite to be honest,
while the Germans are too honest to be polite.

:)

Reply:
> Thanks for the free English lesson :)

Always happy to (over)share the knowledge :stuck_out_tongue: 

> 
> I was under the impression that - exactly because "must" is so strong - it should not be used in English as it can be perceived as impolite and one should use something like "you might consider to do this and that" instead.

That would probably be true if we were in a different setting. 

> 
> The British are too polite to be honest,
> while the Germans are too honest to be polite.

:joy:

And technical writers are the least polite of all. None of those extra words hanging around and messing the place up for us!